timeseries/src/autogluon/timeseries/models/presets.py

```diff
@@ -44,6 +45,7 @@
     DLinear=DLinearModel,
     PatchTST=PatchTSTModel,
     TemporalFusionTransformer=TemporalFusionTransformerModel,
+    WaveNet=WaveNetModel,
```

Comment:
Shouldn't the model also get a default priority and a place in `best` or does that come after testing?

Reply:
I plan to add priority + refactor the presets after running zero-shot HPO for portfolio selection.